pages/content/shared/experiments.html

```diff
@@ -0,0 +1,36 @@
+$view: /layouts/blank.j2
+$sitemap:
+  enabled: false
+---
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="/" />
+  <meta name="viewport" content="width=device-width,minimum-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    {% do doc.styles.addCssFile('css/base.css', -3) %}
+    {% do doc.styles.addCssFile('css/components/atoms/anchor.css', -2) %}
+    {% do doc.styles.addCssFile('/css/components/molecules/experiments.css', -1) %}
+    {{ doc.styles.emit() }}
+    html, body { background: none transparent; }
+    body { display: flex; min-height: 100vh; align-items: center; }
+  </style>
+</head>
+<body>
+  <div class="ap-m-experiments ap-m-experiments-iframe">
+    {% include '/views/partials/experiments/experiments-controls.j2' %}
+    <b class="reload-hint" hidden>{{ _('Please reload the page.' ) }} ↻</b>
+  </div>
+  <script>
+  const experiments = [
```

Comment:
You could use `[= experiments|tojson =]` instead of building the array literally.

Reply:
Thanks for pointing this out, I changed it to
`const experiments = [= experiments|dump|safe =]`

Way better!